core/domain/services/assets/CoreAssetManager.php

```diff
@@ -335,7 +401,7 @@ private function getAccountingSettings() {
 
     /**
      * Returns configuration data for the js Currency VO.
-     * @sinc $VID:$
+     * @since $VID:$
```

Comment:
should probably replace `$VID:$` with `4.9.62.p` here (or whatever other methods around it are using) since this obviously isn't being added in this branch and got skipped when setting versions

Reply:
This was added in the GBM master branch, hence it's still needed as is so when it gets merged to master it gets the appropriate version.

Reply:
Your comment may have been because you thought `@since` is in the pattern matching for `$VID:$` replacement.  It's not.